react/react1/week2/react-week2/src/components/TodoApp.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import {useState } from "react";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faTrash } from '@fortawesome/free-solid-svg-icons'
+// import { seconds } from "./TodoTimer";
+
+const todos = [
+{id: 1, description: "Get out of bed", deadline: "Wed Sep 13 2017" }, 
+{id: 2, description: "Brush teeth", deadline: "Thu Sep 14 2017"}, 
+{id: 3, description: "Eat breakfast", deadline: "Fri Sep 15 2017" },
+{id: 4, description: "Eat launch", deadline: "Fri Sep 16 2017" }
+];
+
+
+export default function Todo() {
+  
+  const [TodoList, addTodo] = useState(todos);
+
+  const addNewTodo = () => {
+    addTodo(current => [...current, 
+      {id: TodoList.length + 1, description: 'Random', deadline: 'Text'}
```

Comment:
in this way, all our items would have the same id, a better way to generate random id can be this: `Math.random().toString(16).slice(2)`